bindings/nodejs/examples/client/05-get-address-balance.ts

```diff
@@ -45,7 +51,9 @@ async function run() {
         });
 
         // Get outputs by their IDs
-        const addressOutputs = await client.getOutputs(outputIdsResponse.items);
+        const addressOutputs = await client.getOutputs(
+            outputIdsResponse.items.map((id) => new OutputId(id)),
```

Comment:
Still needed because outputIdsResponse is an interface

Reply:
Hmm ok, looks like a double wrap to me but if you say so

Reply:
It's because the class is in an interface and then it's not actually this class object, this was also the reason why I initially didn't set this type there